record_adb.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# Common functions used throughout FNPRMS
+
+
+
+
+import sys
+import argparse
+import subprocess
+import time
+import os
+import signal
+
+
+#This script records the phone either through intent or simulating user touch for the purpose of generating videos that are consistent.
+#The consistency allows us to compare said video.
+def main(args):
+    method = args.input
+    recording_name = './sdcard/output.mp4'
+
+    if args.output is not None:
+            recording_name = './sdcard/' + args.output
+    if method in 'touch' and (args.coordinate_x is None or args.coordinate_y is None):
+        print('--touch requires --coordinate-x <coordinate> --coordinate-y <coordinate> to use the touch input')
+        sys.exit()
+    if method in 'intent' and args.package is None:
+        print('--intent requires --package <your.package.name>')
+        sys.exit()
+
+    check_for_existing_process()
```

Comment:
tbh, I would have preferred if you didn't edit the script significantly because now I have more to review and there are higher priority matters for me to get to. I hope that makes sense. :)

Reply:
Yeah sorry about that, I just wanted to make it clean. We can move on. its just for this recording and I guess future use so its clear if ever someone wanted to use it.